src/index.html

```diff
@@ -11,7 +11,164 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form class="Registration"
+     action=" https://mate-academy-form-lesson.herokuapp.com/create-application"
+     method="post"
+     autocomplete="off">
+      <fieldset class="section pinf">
+        <legend>Personal information</legend>
+
+        <label class="form-field surname">
+          Surname:
+          <input type="text" name="attribute Surname">
+        </label>
+
+        <label class="form-field name">
+          Name:
+          <input type="text" name="attribute Name">
+        </label>
+
+        <label class="form-field age">
+          How old are You?
+          <input
+          type="number"
+          name="attribute age"
+          min="1" max="100"
+          value="12"
+          >
```

Comment:
fix all such cases in the code
<img width="1223" alt="image" src="https://user-images.githubusercontent.com/70220667/230761182-af7579b6-eef5-4736-ae7a-585c7abec282.png">

```suggestion
          <input
            type="number"
            name="attribute age"
            min="1" max="100"
            value="12"
          >
```